src/components/PostItem.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Post } from '../types/Post';
+
+type Props = {
+  post: Post,
+  isSelected: boolean,
```

Comment:
Consider renaming the `setSelectedPost` prop to `onSelectPost` to better indicate it's an event handler function 🤖